internal/network/data_sender.go

```diff
@@ -134,7 +143,7 @@ func (ds *dataSender) SendDeviceNetworkData(data models.DeviceNetworkData) error
 		return errors.Wrap(err, "failed to marshall cloudevent")
 	}
 
-	err = ds.sendPayload(deviceNetworkTopic, payload)
+	err = ds.sendPayload(deviceNetworkTopic, payload, false)
```

Comment:
let's also compress this one.. coming up we'll need to create that topic to in dimo-emqx